src/main/java/com/commercetools/project/sync/util/SyncUtils.java

```diff
@@ -89,5 +94,41 @@ public static void logWarningCallback(
         exception);
   }
 
+  @Nonnull
+  public static String buildLastSyncTimestampContainerName(
+      @Nonnull final String syncModuleName, @Nullable final String runnerName) {
+
+    final String syncModuleNameWithLowerCasedFirstChar = lowerCaseFirstChar(syncModuleName);
+    return format(
+        "%s.%s.%s",
+        getApplicationName(),
+        getRunnerNameValue(runnerName),
+        syncModuleNameWithLowerCasedFirstChar);
+  }
+
+  @Nonnull
+  private static String lowerCaseFirstChar(@Nonnull final String string) {
```

Comment:
I guess this one can be done by StringUtils?
https://commons.apache.org/proper/commons-lang/javadocs/api-release/org/apache/commons/lang3/StringUtils.html#uncapitalize-java.lang.String-

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/168/commits/51f64a1f90267414e38c8bf6c251addebf9c1e20

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/168/commits/51f64a1f90267414e38c8bf6c251addebf9c1e20